src/cli.rs

```diff
@@ -86,6 +95,10 @@ pub struct Cli {
     #[arg(long, env, default_value = "text")]
     pub log_format: LogFormat,
 
+    /// Directory to write logs to
+    #[arg(long, env)]
+    pub log_dir: Option<PathBuf>,
```

Comment:
I think this should go inside a default directory as we need to grant perms to write into this FS directory in the dockerfile. This arg could be used for the name of the file instead?